definitions/divorce/json/CaseEventToFields/CaseEventToFields-nonprod.json

```diff
@@ -271,9 +271,9 @@
     "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "EnterAOSOffline_CoRespondent",
-    "CaseFieldID": "CoRespondentSolicitorAddress",
+    "CaseFieldID": "CoRespondentSolicitorName",
     "PageFieldDisplayOrder": 15,
-    "DisplayContext": "OPTIONAL",
+    "DisplayContext": "MANDATORY",
     "PageID": "aosdataentrycoresp1",
     "PageLabel": "Enter scanned form case data",
     "PageDisplayOrder": 1,
```

Comment:
Im a bit out of the loop with these recent stories. What has made you have to change the order of theses Case Events ?

Reply:
Hey Jack, welcome back!
Those are just fields, just changed the order so that the solicitor's name appears after their address rather than before. It's just a matter of preference really

Reply:
Thanks Marc! 
Perfect, that makes a lot of sense!